### ISSUE TITLE
Auto generate helm values.yaml file from toml file

### ISSUE THREAD

**Issue by oap75** [cartesi/helm-charts#79]
## 📚 Context
We need to find a way to auto generate values.yaml from the rollups node [config.toml](https://github.com/cartesi/rollups-node/blob/main/internal/config/generate/Config.toml) file.

## ✔️ Solution
Generate a script or find an existing service to do it.

## 🎯 Definition of Done
Final `values.yaml` file that generated from `config.toml` file.

**Comment by endersonmaia** [comment]
I think this is useful https://github.com/mikefarah/yq/#install

I think we could use the current namespaces inside the `Config.toml` and have something like this:

```yaml
config:
  logging:
    CARTESI_LOG_LEVEL:
    CARTESI_LOG_TIMESTAMP:
  features:
    CARTESI_FEATURE_HOST_MODE:
...
```

Also, `config:` could be inside the `validator:`

```yaml
validator:
  config:
...
```

The main feature is that we can make safe merges between multiple values files, where the latest value file replaces values defined at previous ones.

**Comment by endersonmaia** [comment]
https://mikefarah.gitbook.io/yq/usage/toml

**Comment by oap75** [comment]
Another challenge is related to the repo documentation (README). So, we should be able to auto generate comments on values.yaml and use it to auto generate the README.

**Comment by endersonmaia** [comment]
> Another challenge is related to the repo documentation (README). So, we should be able to auto generate comments on values.yaml and use it to auto generate the README.

Config.toml has documentations fields, we could use them.

https://github.com/cartesi/rollups-node/blob/main/internal/config/generate/Config.toml#L11-L12

**Comment by oap75** [comment]
Fixed by version 1.3.0.
